packages/e2e/artillery/artillery.ts

```diff
@@ -0,0 +1,16 @@
+export interface Context<T> {
+  _uid: string;
+  vars: T;
+}
+
+export type AfterResponse<T> = (
+  params: unknown,
+  response: unknown,
+  ctx: Context<T>,
+  ee: unknown,
```

Comment:
What's `ee`?

Reply:
If read correctly the artillery source code should be some event emitter... but I didn't found what it actually is.
The callbacks receive it as an argument, probably I didn't investigated its meaning/purpose as I was able to catch the POC target without using it.

Reply:
I confirm, it is the `EventEmitter` and it's used to collect custom metrics (POC added).